Sources/Swift/Integrations/SessionReplay/SentryReplayRecording.swift

```diff
@@ -0,0 +1,34 @@
+@_implementationOnly import _SentryPrivate
+import Foundation
+
+@objcMembers
+class SentryReplayRecording: NSObject {
+    
+    static let SentryReplayEncoding = "h264"
+    static let SentryReplayContainer = "mp4"
+    static let SentryReplayFrameRateType = "constant"
+    
+    let segmentId: Int
+
+    let meta: SentryRRWebMetaEvent
```

Comment:
Should probably be a list of `SentryRRWebEvent` maybe? You'd have to maintain it for each type of rrweb event otherwise

Reply:
Sure, a list works too.